setup.py

```diff
@@ -50,11 +20,11 @@ def run_tests(self):
 DEPENDENCIES = [
     "matplotlib>=3.2.0",
     "Pillow>=6.2.2",  # not our dependency but older versions crash with mpl
-    "numpy>=1.18.0",
-    "pandas==0.25.*",
-    "scipy>=1.4.0",
-    "scikit-learn>=0.22.0",
-    "tfs-pandas>=1.0.3",
+    "numpy>=1.19.0",
+    "pandas>=1.0",
```

Comment:
Do really we need to force pandas `>=1.0`?

Leaving `>=0.25` instead will let `pip` install `1.x` which we're compatible with but give some more freedom in environments.

Reply:
But new tfs requirements are also >=1.0 so  i think there is nothing to be gained there as well, is there?

Reply:
I guess you're right. @lmalina do you approve?

Reply:
we need pandas >=1.0 because of .convert_dtypes()